### ISSUE TITLE
Extra Credit: Trump Email

### ISSUE THREAD

**Issue by mikeizbicki** [mikeizbicki/cmc-csci143#368]
You can get 1 point of extra credit if you send me an email from `trump@whitehouse.gov` requesting it.

**Background:** Email servers use the [Simple Mail Transfer Protocol (SMTP)](https://en.wikipedia.org/wiki/Simple_Mail_Transfer_Protocol) to send emails.  It turns out that SMTP is a fundamentally insecure protocol because sending an email does not require a password.  It is therefore "easy" to do [email spoofing](https://en.wikipedia.org/wiki/Email_spoofing) where you send an email from someone else's account.  The Claremont Colleges implement a number of countermeasures such as [DKIM](https://en.wikipedia.org/wiki/DomainKeys_Identified_Mail) and [SMTP Auth](https://en.wikipedia.org/wiki/SMTP_Authentication), but it is still technically possible to circumvent these countermeasures.  To complete this extra credit assignment, you'll have to research the SMTP protocol and figure out how to circumvent these countermeasures.

**tl;dr** NEVER TRUST THAT THE SENDER OF AN EMAIL IS WHO THEY SAY THEY ARE.
